Add ishpreet to Contributors list

ishpreet95 opened the pull request.

ishpreet95 committed:
Add ishpreet to Contributors list

dA505819 reviewed (changes requested):
Resolve the merge conflict.

ishpreet95 commented:
it is because of prettier formatting, isn't it?

ishpreet95 committed:
resolved the conflicts

I guess it all happened because of the prettier formatting. Let me know if it is all right now. 😊

ishpreet95 committed:
Update Contributors.md

ishpreet95 committed:
trying to resolve conflicts.

eliminated a trailing whitespace as indicated in the conflicts section.

ishpreet95 commented:
hopefully, it's all good now. 🙄

ishpreet95 commented:
I think I will have to close this pull request and push it again without using prettier. Please tell me what to do here.

ishpreet95 commented:
Ahh, good times.
